lib/js/components/Buttons/IconButton/IconButton.vue

```diff
@@ -113,14 +160,53 @@
 			display: initial;
 		}
 
-		&.-minor {
-			@include iconButtonColor($color-minor);
+		&.-neutral {
+			color: $color-neutral-text !important;
```

Comment:
@damianmef I looked at it. You could exclude these classes in the loop, but I'd also choose `important`

Reply:
I'm also not a fan of `important` and use it in specially situations, where it saves a lot of code and probably will not interfere with other rules in the future. If any `important` will make problems in the future, it should be rebuild. Ok?